CHANGELOG.md

```diff
@@ -3,7 +3,22 @@
 The present file will list all changes made to the project; according to the
 [Keep a Changelog](http://keepachangelog.com/) project.
 
-## [9.4] unreleased
+## [9.4.1] unreleased
+
+### Added
```

Comment:
Well, we may not have any add or change... Maybe it is useless to put them right now ;) (even if I understand why you didi)